elevenlabs/src/assets/agent-config.json

```diff
@@ -148,7 +148,7 @@
             "dynamicVariables": {
               "dynamicVariablePlaceholders": {}
             },
-            "executionMode": "immediate",
+            "executionMode": "async",
             "apiSchema": {
               "url": "https://n8n-webhooks-home-assistant.ffmathy.org/webhook/13ed97a5-c9fe-4815-ab6c-7b9b1b42593c",
```

Comment:
This field should be removed when refreshing the agent configuration, as it is considered secret. Also remove all past history associated with this change.

Reply:
It's okay to do the disruptive work.